src/adagrams.js

```diff
@@ -1,8 +1,97 @@
+const poolOfLetters = ( 
+  "A".repeat(9) + 
+  "B".repeat(2) + 
+  "C".repeat(2) + 
+  "D".repeat(4) + 
+  "E".repeat(12) + 
+  "F".repeat(2) + 
+  "G".repeat(3) + 
+  "H".repeat(2) + 
+  "I".repeat(9) + 
+  "J".repeat(1) + 
+  "K".repeat(1) + 
+  "L".repeat(4) + 
+  "M".repeat(2) + 
+  "N".repeat(6) + 
+  "O".repeat(8) + 
+  "P".repeat(2) + 
+  "Q".repeat(1) + 
+  "R".repeat(6) + 
+  "S".repeat(4) + 
+  "T".repeat(6) + 
+  "U".repeat(4) + 
+  "V".repeat(2) + 
+  "W".repeat(2) + 
+  "X".repeat(1) + 
+  "Y".repeat(2) + 
+  "Z".repeat(1)
+  ).split('');
+
+const valueHash = {"A": 1, "E": 1, "I": 1, "O": 1, "U": 1, "L": 1, "N": 1, "R": 1, "S": 1, "T": 1, "D": 2, "G": 2, "B": 3, "C": 3, "M": 3, "P": 3, "F": 4, "H": 4, "V": 4, "W": 4, "Y": 4, "K": 5, "J": 8, "X": 8, "Q": 10, "Z": 10};
```

Comment:
style note: each letter should be on it's own line.
In addtion, this notation
`const valueHash = {A: 1, E: 1, ... }`
will create
`const valueHash = {"A": 1, "E": 1, ... }`
`